src/devices/Bmp180/Bmp180.cs

```diff
@@ -0,0 +1,304 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+//Ported from https://github.com/adafruit/Adafruit_Python_BMP/blob/master/Adafruit_BMP/BMP085.py  
+//Formulas and code examples can also be found in the datasheet https://cdn-shop.adafruit.com/datasheets/BST-BMP180-DS000-09.pdf
+
+using System;
+using System.Device.I2c;
+using System.Buffers.Binary;
+using System.Threading;
+
+namespace Iot.Device.Bmp180
+{
+    public class Bmp180:IDisposable
+    {
+        private I2cDevice _i2cDevice;
+        private readonly CommunicationProtocol _communicationProtocol;
+        private bool _initialized = false;
+        private readonly CalibrationData _calibrationData;
+        private Sampling _mode = Sampling.Standard;
+
+        private enum CommunicationProtocol
+        {
+            I2c

```

Comment:
Does this device support SPI as well? if not, then I don't really see the point of having the CommunicationProtocol enum.

Reply:
I see, in that case, for the time being I would remove this part of the communication protocol, since as of now we only have one, and add this back whenever we add support for SPI as well.